database/migrations/2015_09_14_135258_create_channels_table.php

```diff
@@ -26,6 +28,6 @@ public function up()
      */
     public function down()
     {
-        Schema::drop('password_resets');
+        //
```

Comment:
who worked on it?. I haven't accepted anyone's pull request yet, you're the first.